docs/semgrep-assistant/customize.md

```diff
@@ -26,18 +26,12 @@ Semgrep recommends setting a low confidence level since even incorrect suggestio
 
 ## Auto-triage
 
-[Auto-triage](/semgrep-assistant/overview/#auto-triage) allows you to get notifications whenever Assistant indicates a finding may be safe to ignore. You can enable auto-triage, if it isn't already, as follows:
-
-1. Sign in to Semgrep AppSec Platform, and navigate to **Settings > Deployment**.
-2. In the **Assistant** section, click the **Auto-triage for Code** <i class="fa-solid fa-toggle-large-on"></i> if it is not yet enabled.
-3. Select whether you want alerts included in your **PR/MR comments** and **Slack notifications**.
+[Auto-triage](/semgrep-assistant/overview/#auto-triage) allows you to get notifications whenever Assistant indicates a finding may be safe to ignore. This feature is available by default once you [enable Assistant](/semgrep-assistant/getting-started). 
```

Comment:
is it available by default or _enabled_ by default? if I'm correct that Assistant just Does The Thing when it's turned on, we might want to be explicit about that so people aren't looking for a switch to turn it on (it's me I'm people)

Reply:
Yeah, once you have Assistant enabled, you have auto-triage -- so you have to turn the Assistant switch on, but you don't have a subsequent auto-triage switch that needs to be turned on. That's what I was trying to convey by saying "available by default" but maybe I muddied the water?

Reply:
That's fair. I think I'll create a new KB section for Assistant and add all of our current troubleshooting information there. Then, I'll remove this section. Expect this shortly!